tests/integration/eZ/API/RichTextFieldTypeIntegrationTest.php

```diff
@@ -624,6 +626,114 @@ public function testConvertRemoteObjectIdToObjectId($test, $expected)
         );
     }
 
+    /**
+     * @throws \eZ\Publish\API\Repository\Exceptions\ContentFieldValidationException
+     * @throws \ErrorException
+     * @throws \eZ\Publish\API\Repository\Exceptions\BadStateException
+     * @throws \eZ\Publish\API\Repository\Exceptions\ContentTypeFieldDefinitionValidationException
+     * @throws \eZ\Publish\API\Repository\Exceptions\ContentValidationException
+     * @throws \eZ\Publish\API\Repository\Exceptions\InvalidArgumentException
+     * @throws \eZ\Publish\API\Repository\Exceptions\NotFoundException
+     * @throws \eZ\Publish\API\Repository\Exceptions\UnauthorizedException
+     */
+    public function testExternalLinkCorrectStoreAfterUpdate()
+    {
+        $xmlDocument = $this->createXmlDocumentWithExternalLink();
+        $repository = $this->getRepository();
+        $contentService = $repository->getContentService();
+        $contentTypeService = $repository->getContentTypeService();
+        $locationService = $repository->getLocationService();
+
+        $contentTypeCreateStruct = $contentTypeService->newContentTypeCreateStruct('test-RichText');
+        $contentTypeCreateStruct->mainLanguageCode = 'eng-GB';
+        $contentTypeCreateStruct->names = ['eng-GB' => 'Test'];
+        $contentTypeCreateStruct->creatorId = $repository->getPermissionResolver()->getCurrentUserReference()->getUserId();
+        $contentTypeCreateStruct->creationDate = $this->createDateTime();
+
+        $fieldDefinitionCreateStruct = $contentTypeService->newFieldDefinitionCreateStruct('description', 'ezrichtext');
+        $fieldDefinitionCreateStruct->names = ['eng-GB' => 'Title'];
+        $fieldDefinitionCreateStruct->fieldGroup = 'main';
+        $fieldDefinitionCreateStruct->position = 1;
+        $fieldDefinitionCreateStruct->isTranslatable = true;
+        $contentTypeCreateStruct->addFieldDefinition($fieldDefinitionCreateStruct);
+
+        $contentGroup = $contentTypeService->loadContentTypeGroupByIdentifier('Content');
+        $contentTypeDraft = $contentTypeService->createContentType($contentTypeCreateStruct, [$contentGroup]);
+
+        $contentTypeService->publishContentTypeDraft($contentTypeDraft);
+        $testContentType = $contentTypeService->loadContentType($contentTypeDraft->id);
+
+        // Create a folder for tests
+        $contentTypeCreateStruct = $contentService->newContentCreateStruct(
+            $contentTypeService->loadContentTypeByIdentifier('folder'),
+            'eng-GB'
+        );
+
+        $contentTypeCreateStruct->setField('name', 'Folder Link');
+        $draft = $contentService->createContent(
```

Comment:
```suggestion
        $folderDraft = $contentService->createContent(
```

Reply:
Side question: do we really need to create a folder for test content here?

Reply:
yes, you're right, we don't have to create a folder, we can work in main location